.ddev/config.yaml

```
@@ -0,0 +1,256 @@
+name: ddev.com
+type: laravel
```

I think `php` will be much better than `laravel`. `php` is essentially "generic"

```suggestion
type: php
```